### DISCUSSION TITLE
Is there any way to deal with static files?

### DISCUSSION THREAD

**Discussion by Lewisyixin** [apache/apisix#5572] [Q&A] [answered]
like '
location / {
     root /www/root/html/;
}'

If not, how does everyone solve such needs? Is there any best practices about it ?
Maybe "deploy a nginx just keep all static files after apisix"  is  a good way?

**Comment by tokers** [comment, marked as answer]
@Lewisyixin You can inject your own Nginx configurations to APISIX, see http://apisix.apache.org/docs/apisix/next/customize-nginx-configuration/.
